pycbc/waveform/ringdown.py

```diff
@@ -485,7 +711,18 @@ def multimode_base(input_params, domain, freq_tau_approximant=False):
         n overtones in the chosen domain (time or frequency).
     """
     input_params['lmns'] = format_lmns(input_params['lmns'])
-    amps, phis = lm_amps_phases(**input_params)
+    amps, phis, dbetas, dphis = lm_amps_phases(**input_params)
+    pols, polnms = lm_arbitrary_harmonics(**input_params)
+    # get harmonics argument
+    try:
+        harmonics = input_params['harmonics']
+    except KeyError:
+        harmonics = 'spherical'
+    # add inclination and azimuthal if they aren't provided
+    if 'inclination' not in input_params:
```

Comment:
This is where the warning or error suggested in my previous comment should go, to make sure the user is aware if what happens if they forget to provide an inclination and azimuth angle.